.circleci/config.yml

```diff
@@ -118,19 +123,26 @@ jobs:
           command: |
             npm install
             npm link
+          environment:
+            NPM_CONFIG_PREFIX: /home/node/.npm-global
```

Comment:
Why is this needed?

Reply:
I've created this CI based on the changes to the logging-winston library, and the that library has the `NPM_CONFIG_PREFIX` environment variable  set in various places.

Reply:
`npm link` will try to change system directory (somewhere in `/usr` and it will fail on Circle when running as non-root, hence we set `NPM_CONFIG_PREFIX`).